test/features/parsoid/transform.js

```diff
@@ -1,8 +1,8 @@
 'use strict';
 
-const assert = require('../../utils/assert.js');
 const Server = require('../../utils/server.js');
-const preq   = require('preq');
+const { REST, assert } = require('api-testing');
```

Comment:
Given that we still need to run CI with node 6, which doesn't support async/await, you can surround this with try-catch and just return and skip this whole test suite if api-testing is not loadable.

Reply:
A try-catch block has been added as per your advice.